kdc/connect.c

```diff
@@ -1122,11 +1127,9 @@ start_kdc(krb5_context context,
     if (max_kdcs < 1)
 	max_kdcs = 1;
 
-    pids = malloc(max_kdcs * sizeof(*pids));
```

Comment:
I would prefer that this block be left alone and that an empty slot always be (pid_t)-1 instead of  0 or -1.